packages/test-utils/src/error-wrapper.js

```diff
@@ -1,57 +1,72 @@
 // @flow
 
 import { throwError } from 'shared/util'
+import { REF_SELECTOR } from 'shared/consts'
+
+const buildSelectorString = (selector: Selector) => {
+  if (selector.type === REF_SELECTOR) {
+    return `ref="${selector.value.ref}"`
+  }
+
+  if (typeof selector.value === 'string') {
+    return selector.value
+  }
+
+  return 'Component'
+}
 
 export default class ErrorWrapper implements BaseWrapper {
-  selector: string
+  selector: Selector
```

Comment:
This is a breaking change. Are we ok with that?

Reply:
Yes, there are big breaking changes in the next release anyway (removing sync mode)